src/browser/appwindow.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
+var _ = require('underscore-plus');
+var BrowserWindow = require('browser-window');
 var dialog = require('dialog');
-var path = require('path');
-var url = require('url');
 var EventEmitter = require('events').EventEmitter;
-var BrowserWindow = require('browser-window');
 var fork = require('child_process').fork;
-var _ = require('underscore-plus');
 var killChildProcess = require('./util/kill-childprocess');
+var path = require('path');
+var shell = require('shell');
```

Comment:
Please follow the "require(s) best practices" by @zemirco
```
// 1. core modules
var fs = require('fs');
var shell = require('shell');

// 2. public modules from npm
var express = require('express');
var uuid = require('node-uuid');

// 3. your own modules from file
var myMod = require('./mod.js');
```
http://www.mircozeiss.com/node-js-require-s-best-practices/